omnibus/package-scripts/agent-deb/postinst

```diff
@@ -8,6 +8,8 @@ INSTALL_DIR=/opt/datadog-agent
 LOG_DIR=/var/log/datadog
 CONFIG_DIR=/etc/datadog-agent
 SERVICE_NAME=datadog-agent
+RUN_DIR=/opt/datadog-agent/run
+SITE_PACKAGES_DIR="${INSTALL_DIR}/embedded/lib/python*/site-packages"
```

Comment:
With RPM/DEB, you can list files that came from the package and check whether they've been modified, so I'm not sure what it would bring us to further separate these. Could you elaborate on why you think it would be better?

Reply:
Sorry for the late reply. You're right, making this distinction does bring much and we should probably rather go with following the same rules for `${INSTALL_DIR}/embedded/bin`

Reply:
@lebauce have you ended up implementing this change? I might be looking wrong, but I don't see it. I think this is the one last thing that needs to be addressed, so if you could do that and rebase the PR, I could approve and merge.

Reply:
@bkabrda Sorry for the delay and thank pinging me about it. I modified the PR to follow the same rules for ${INSTALL_DIR}/embedded/bin
It seems to pass the full (`inv pipeline --deploy`) test suite